sdcflows/utils/tests/test_tools.py

```diff
@@ -23,11 +23,50 @@
 """Test EPI manipulation routines."""
 import numpy as np
 import nibabel as nb
-from ..tools import brain_masker
+from sdcflows.utils.tools import brain_masker, deoblique_and_zooms
 
 
 def test_epi_mask(tmpdir, testdata_dir):
     """Check mask algorithm."""
     tmpdir.chdir()
     mask = brain_masker(testdata_dir / "epi.nii.gz")[-1]
     assert abs(np.asanyarray(nb.load(mask).dataobj).sum() - 166476) < 10
+
+
+def test_deoblique_and_zooms():
+    """Check deoblique and denser."""
+
+    # Generate an example reference image
+    ref_data = np.zeros((20, 30, 40), dtype=np.float32)
+    ref_affine = np.eye(4)
+    ref_affine[:3, :3] = np.diag([2, 3, 4])  # Set zooms to (2, 3, 4)
+    ref_img = nb.Nifti1Image(ref_data, ref_affine)
+
+    # Generate an example oblique image
+    ob_data = np.zeros((25, 35, 45), dtype=np.float32)
+    ob_affine = np.eye(4)
+
+    # Rotate 90 degrees around x-axis
```

Comment:
This test seems fine to test, but I would also add a 30deg test.

Reply:
Yes, I was too lazy to pre-calculate what the shapes must result for several padding/factor combinations so opted for looking at the volume after interpolation. I have to check why there seems to be a substantial error when doing that. Once I figure that out, a 30 deg test (or whatever) could be added with parametrization (and tested with factors/paddings)

Reply:
Added some parametric tests.